### PR TITLE
docs/16060-compare-cumulative-articles

### PR DESCRIPTION
**Pull request by raf18seb**
Added general documentation articles for `compare` and `cumulative` features. See #16060

### CODE REVIEW

File: docs/stock/compare.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Compare
+================
+
+The compare tool is used to compare the values to the first value in a visible range.
```
Comment (karolkolodziej) [inline docs/stock/compare.md:4]: I would change it to:
`The compare tool is used to compare each points' value to the first value in a visible range.`
It's not exactly clear what we are comparing :)

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
Besides one improvement it looks really good!
Great pictures, simple examples to calculate the math.
Good job ;)

File: docs/stock/compare.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+![compare-percent.png](compare-percent.png)
+
+The `compare` can be enabled in the chart's options, see the [API reference](https://api.highcharts.com/highstock/plotOptions.series.compare). Available options for the `series.compare` property are:
+*   `percent` - percentage difference (image above), e.g. if the first visible value is 10 and the currect value is 8, then the compare equals to 80% (or -20% when the [compareBase](https://api.highcharts.com/highstock/plotOptions.series.compareBase) is set to 0).
+*   `value` - difference between the values, e.g. if the first visible value is 10 and the currect value is 8, then the compare equals to -2.
```
Comment (pawelfus) [inline docs/stock/compare.md:10]: currect -> current

File: docs/stock/compare.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+The `compare` can be also enabled/disabled on a specific series by the [series.setCompare()](https://api.highcharts.com/class-reference/Highcharts.Series#setCompare) method or on all the series belonging to a specific y-axis by the [axis.setCompare()](https://api.highcharts.com/class-reference/Highcharts.Axis#setCompare) method.
```
Comment (pawelfus) [inline docs/stock/compare.md:12]: `axis.setCompare()` -> `yAxis.setCompare()`

File: docs/stock/compare.md (hunk 1 above)
Comment (pawelfus) [reply docs/stock/compare.md:4]: PS: We also have [`compareStart`](https://api.highcharts.com/highstock/series.line.compareStart) option :)

File: docs/stock/cumulative-sum.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Cumulative Sum
+================
+
+The Cumulative Sum tool sums (cumulates) all the previous values with the current value in a visible range.
+
+For the following data: `[10, 8, 15, 20, 8, 15]` the Cumulative Sum returns `[10, 18, 33, 53, 61, 76]` (see the graph below):
+![cumulative-sum.png](cumulative-sum.png)
+
+The `cumulative` can be enabled in the chart's options using the [series.cumulative](https://api.highcharts.com/highstock/plotOptions.series.cumulative) property or enabled/disabled by the [series.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Series#setCumulative) method or on all the series belonging to a specific y-axis by the [axis.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Axis#setCompare) method.
```
Comment (pawelfus) [inline docs/stock/cumulative-sum.md:9]: `axis.setCumulative()` -> `yAxis.setCumulative()`

**Review by pawelfus — CHANGES_REQUESTED** [review]
Looks good, just some minor changes

File: docs/stock/cumulative-sum.md [hunk 5]
```diff
@@ -6,4 +6,4 @@ The Cumulative Sum tool sums (cumulates) all the previous values with the curren
 For the following data: `[10, 8, 15, 20, 8, 15]` the Cumulative Sum returns `[10, 18, 33, 53, 61, 76]` (see the graph below):
 ![cumulative-sum.png](cumulative-sum.png)
 
-The `cumulative` can be enabled in the chart's options using the [series.cumulative](https://api.highcharts.com/highstock/plotOptions.series.cumulative) property or enabled/disabled by the [series.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Series#setCumulative) method or on all the series belonging to a specific y-axis by the [axis.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Axis#setCompare) method.
\ No newline at end of file
+The `cumulative` can be enabled in the chart's options using the [series.cumulative](https://api.highcharts.com/highstock/plotOptions.series.cumulative) property or enabled/disabled by the [series.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Series#setCumulative) method or on all the series belonging to a specific y-axis by the [yAxis.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Axis#setCompare) method.
```
Comment (karolkolodziej) [inline docs/stock/cumulative-sum.md:9]: `[yAxis.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Axis#setCompare)`

Here the `setCumulative` leads to the `setCompare`

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**